utils/dbInspect.js

```diff
@@ -354,14 +367,34 @@ async function formatCDS(db, object, fields, constraints, type, parent) {
 	if (type === "view" || type === "table") {
 		cdstable += "@cds.persistence.exists \n"
 	}
-	if (type === "view" || type === "hdbview") {
-		object[0].VIEW_NAME = object[0].VIEW_NAME.replace(/\./g, "_");
-		object[0].VIEW_NAME = object[0].VIEW_NAME.replace(/:/g, "");
```

Comment:
So new code seems to be 99% compatible except this line. Why do we have this special case for views? 
in case if it's a table `a.b.c::d.e` we had `a_b_c::d_e` but for views it was `a_b_cd_e`. So I propose to normalize the approach and make the rule straightforward: 
by default old logic is applied with `. -> _` rule. Using flags we can control renaming 
@jung-thomas

Reply:
There is some special handling here that I'd done for the hana-cli cds command.  I needed some escaping when dynamically generating the cds metadata for the cds serve and request handling. I'll go ahead and merge this and then double check testing locally that we aren't breaking anything I expect over in this other command.